### PR TITLE
Edit Item's license

### PR DESCRIPTION
**Pull request by milanmajchrak**
| Phases            | MP | MM  |  MB  | MR |   JM | Total  |
|-----------------|----:|----:|----:|-----:|-----:|-------:|
| ETA                  |  0  |  0  |    0 |     0 |      0 |        0 |
| Developing      |  0  |  0  |    0 |    0 |      0 |         0 |
| Review             |  0  |  0  |    0 |    0 |      0 |         0 |
| Total                |   -  |   -  |   -   |  -    |   -    |         0 |
| ETA est.             |      |      |       |       |         |         0 |
| ETA cust.           |   -  |   -  |   -  |   -   |   -     |        0 |
## Analysis
(Write here, if there is needed describe some specific problem. Erase it, when it is not needed.)
## Problems
(Write here, if some unexpected problems occur during solving issues. Erase it, when it is not needed.)

### CODE REVIEW

**Review by MajoBerger — APPROVED** [review]

File: src/assets/i18n/cs.json5 [hunk 1]
```diff
@@ -2469,6 +2469,28 @@
...
+  // "item.edit.tabs.license.head": "License",
+  "item.edit.tabs.license.head": "Licence",
+  // "item.edit.license.head": "Current License",
+  "item.edit.license.head": "Stávající licence",
+  // "item.edit.license.select.license.initial-value.message": "-Select License-",
+  "item.edit.license.select.license.initial-value.message": "-Vyberte licenci-",
+  // "item.edit.license.select.license.select.message": "Select a license",
+  "item.edit.license.select.license.select.message": "Vyberte licenci",
+  // "item.edit.license.select.license.info": "If you want to change the license, select a new license from the list and press update.",
+  "item.edit.license.select.license.info": "Přejete-li si změnit licenci, vyberte ji ze seznamu a zmáčkněte změnit.",
```
Comment (vidiecan) [inline src/assets/i18n/cs.json5]: zmáčkněte ?

Comment (vidiecan) [reply src/assets/i18n/cs.json5]: klikněte 
ale co je to "zmenit"?

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: src/assets/i18n/cs.json5 (hunk 1 above)
Comment (milanmajchrak) [reply src/assets/i18n/cs.json5]: I copied all translations from the current Lindat page. Should I change it?

Comment (vidiecan) [reply src/assets/i18n/cs.json5]: do we have a button called "zmenit"?

**Review by vidiecan — CHANGES_REQUESTED** [review]

File: src/assets/i18n/cs.json5 (hunk 1 above)
Comment (milanmajchrak) [reply src/assets/i18n/cs.json5]: Changed to `zmáčkněte aktualizovat`

**Review by vidiecan — APPROVED** [review]

**Pull request merged**